musicbot/bot.py

```diff
@@ -662,6 +679,19 @@ def _autopause(player):
                 await self.change_presence(activity=game)
                 self.last_status = game
 
+    async def update_now_playing_topic(self, player, entry=None, is_paused=False):
+        guild = player.voice_client.guild
+        last_np_msg = self.server_specific_data[guild]['last_np_msg']
+        channel = last_np_msg.channel if last_np_msg else None
```

Comment:
It would likely be better if there was an additional option to specify a channel instead of using the last now playing message.

Reply:
You're right, it is implemented now.